File: CursoPOOUber/PHP/car.php
<?php	
require_once('account.php');
class Car 
{
    public $id = integer;
    public $license = string;
    public $driver = string;
    public $passegenger = string;

    public function __construct($license, $driver){
        $this->license -> $license;
        $this->driver -> $driver;
    }

    public function printDataCar(){
        echo"Licencia: $this->license Driver: ".$this->driver->name;
    }
}
?>

File: CursoPOOUber/Java/Car.java
package Java;

class Car {
    private Integer id;
    private String license;
    private Account driver;
    protected Integer passegenger;

    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;        
    }

    void printDataCar(){
        if(passegenger != null){
            System.out.println("License: " + license + " Name Driver: " + driver.name  + " Passengers: " + passegenger);
        }
        
    }

    public Integer getPassanger(){
        return passegenger;
    }

    public void setPassanger(Integer passanger) {
        if(passanger == 4){
            this.passegenger = passanger;
        }else{
            System.out.println("Necesitas asignar 4 pasajeros");
        }
        
    }
}
